rollup.app.js

```diff
@@ -23,7 +23,7 @@ export default {
       // we'll extract any component CSS out into
       // a separate file — better for performance
       css: css => {
-        css.write("public/bundle.css");
+        css.write("bundle.css");
```

Comment:
Why do we write to a different location?

Reply:
This was a [major breaking change](https://github.com/sveltejs/rollup-plugin-svelte/blob/master/CHANGELOG.md#600) in `svelte-rollup`. Now the path is relative to the output of the bundle.